lib/livebook/hubs/team.ex

```diff
@@ -109,7 +109,16 @@ defimpl Livebook.Hubs.Provider, for: Livebook.Hubs.Team do
   @public_key_prefix Livebook.Hubs.Team.public_key_prefix()
 
   def load(team, fields) do
-    struct(team, fields)
+    {offline, fields} = Map.pop!(fields, :offline)
+
+    offline =
+      if offline do
+        :persistent_term.get({__MODULE__, :offline, fields.id})
```

Comment:
```suggestion
        :persistent_term.get({__MODULE__, :offline, fields.id}, nil)
```

And then we don't need the migration?